### PR TITLE
adds option to sort by and order by

### PR DESCRIPTION
**Pull request by saeidsaeity**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -15,14 +15,14 @@
```
Comment (nc-saima-khanom) [inline src/App.css:15]: just as an FYI the tab name in the browser for your app still says rare treasures?

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -1,22 +1,31 @@
 import { useEffect,useState } from "react"
 import { getArticles } from "./utils/api"
 import Grid from '@mui/material/Unstable_Grid2'; 
-import { Link, useSearchParams } from "react-router-dom";
+import { Form, Link, useSearchParams } from "react-router-dom";
 import Topics from "./Topics";
 
 function Articles() {
```
Comment (nc-saima-khanom) [inline src/components/Articles.jsx:7]: your default sort when you change from one sort query to another is descending, even if the previous query was ascending, and the user doesnt change this. E.g. when you load the page, the options say date and ascending, but the articles are sorted by descending date. When i change this to descending then ascending, thats when the order changes to ascending. now if i change the sort by to author, and leave the ascending, it sorts by author but the default is reset to descending even though your app is telling me its sorted by ascending order. 
In summary you need to do two things - make sure your default view of the asc/desc options is set to desc and not asc. And when a user changes the order by to asc this should persist when they change the sort by to another option. please let me know if you have any questions about this!
